### ISSUE TITLE
Unable to disable subtitles added on video

### ISSUE THREAD

**Issue by keukeu1** [0x776b7364/toggle.sg-download#25]
Hi,

Many thanks for your wonderful tool. I'm having some small issues downloading this particular series: Tiger Mum. The download is smooth and is able to complete however there's one problem which is I do not want the sub-titles being added into the video. It seems like a hard sub being added into the video as I'm unable to disable the subtitles when I play the video. However on the toggle website, there's a option to disable the subtitles. Do you have a solution for this issue? Thank you.

**Comment by 0x776b7364** [comment]
Are you able to provide a link to the series? I haven't observed that behaviour before.

**Comment by keukeu1** [comment]
Hi. this is the link to the series. http://tv.toggle.sg/en/shows/tiger-mum-hd/episodes

**Comment by keukeu1** [comment]
Okay, just an update. some of the newer series have similar issues too. Download is able to complete as usual but the subtitles issues is present in the video which can be disabled in the website itself however when playing the downloaded file, subtitles cannot be disabled.

**Comment by peterhoeg** [comment]
When you download that series you get the subtitles in a standalone .srt file as expected but in addition, the same subs are hardcoded in the video. Maybe there are 2 streams (one with and one without hardcoded subs) and the script is simply choosing the wrong one?

**Comment by keukeu1** [comment]
Yup. This is basically the issue. Was hoping to get the one without the hardcoded subs instead. Hopefully there will be some solutions to this as most of the newer dramas are having such issues too.

**Comment by 0x776b7364** [comment]
I've observed the behaviour you've described as well. I'll look into that soon.

**Comment by keukeu1** [comment]
ahh :) thank you.

**Comment by 0x776b7364** [comment]
Weirdly, when I accessed, for e.g., episode 1 of Tiger Mum using Chrome, even though I see the Subtitle settings, I couldn't make the English subtitles appear on-screen. The stream names always have the 'SUB' string, so I'm inclined to think that ffmpeg might be combining the video and the subtitle stream automatically. I'm a little out of ideas, so any recommendations are welcome.

**Comment by icemanfusion** [comment]
I'm facing the same issues too with the drama, Good Luck which is strange because previously when I downloaded one of the episodes, the subtitles doesn't seem to be hard-subbed (was using the  old script owner posted somewhere earlier this year around January or so) @0x776b7364 do you have an earlier version of your script?

**Comment by 0x776b7364** [comment]
For earlier versions, you can check out the file's History on Github. https://github.com/0x776b7364/toggle.sg-download/commits/master/download_toggle_video2.py

**Comment by icemanfusion** [comment]
Hi thanks. Hmm, doesn't seem to work now. Hoping for a solution for this since the hard-subbed subtitles makes it terrible to watch given how big it is on screen.

**Comment by NGPriest** [comment]
What if you change your preferences in this section?

```
FILE_PREFERENCES = 	[(1,'STB','m3u8'),	# generally 720p, Set-top Box, requires ffmpeg
			(2,'ADD','mp4'),	# generally 540p, Android device
			(3,'IPAD','m3u8'),	# generally 540p, iPad, requires ffmpeg
			(4,'IPH','m3u8')]	# generally 360p, iPhone, requires ffmpeg```
